Tools/Pipeline/Controls/PropertyGridTable.cs

```diff
@@ -196,7 +196,7 @@ private void Drawable_Paint(object sender, PaintEventArgs e)
             if(_edit)
             {
                 if (!Global.Unix)
-                    _selectedCell.Edit(pixel1);
+                    _selectedCell?.Edit(pixel1);
```

Comment:
Also that is a characteristic of C# 6.0, but the build bot is using the C# 5.0 compiler